core/src/themes/ionic.theme.default.scss

```diff
@@ -94,6 +94,53 @@ $background-color-rgb:              var(--ion-background-color-rgb, $background-
+  --ion-background-color-step-450: #{mix($text-color-value, $background-color-value, 45%)};
+  --ion-background-color-step-500: #{mix($text-color-value, $background-color-value, 50%)};
+  --ion-background-color-step-550: #{mix($text-color-value, $background-color-value, 55%)};
+  --ion-background-color-step-600: #{mix($text-color-value, $background-color-value, 60%)};
+  --ion-background-color-step-650: #{mix($text-color-value, $background-color-value, 65%)};
+  --ion-background-color-step-700: #{mix($text-color-value, $background-color-value, 70%)};
+  --ion-background-color-step-750: #{mix($text-color-value, $background-color-value, 75%)};
+  --ion-background-color-step-800: #{mix($text-color-value, $background-color-value, 80%)};
+  --ion-background-color-step-850: #{mix($text-color-value, $background-color-value, 85%)};
+  --ion-background-color-step-900: #{mix($text-color-value, $background-color-value, 90%)};
+  --ion-background-color-step-950: #{mix($text-color-value, $background-color-value, 95%)};
+
+  --ion-text-color-step-50: #{mix($background-color-value, $text-color-value, 5%)};
+  --ion-text-color-step-100: #{mix($background-color-value, $text-color-value, 10%)};
+  --ion-text-color-step-150: #{mix($background-color-value, $text-color-value, 15%)};
+  --ion-text-color-step-200: #{mix($background-color-value, $text-color-value, 20%)};
+  --ion-text-color-step-250: #{mix($background-color-value, $text-color-value, 25%)};
+  --ion-text-color-step-300: #{mix($background-color-value, $text-color-value, 30%)};
+  --ion-text-color-step-350: #{mix($background-color-value, $text-color-value, 35%)};
+  --ion-text-color-step-400: #{mix($background-color-value, $text-color-value, 40%)};
+  --ion-text-color-step-450: #{mix($background-color-value, $text-color-value, 45%)};
+  --ion-text-color-step-500: #{mix($background-color-value, $text-color-value, 50%)};
+  --ion-text-color-step-550: #{mix($background-color-value, $text-color-value, 55%)};
+  --ion-text-color-step-600: #{mix($background-color-value, $text-color-value, 60%)};
+  --ion-text-color-step-650: #{mix($background-color-value, $text-color-value, 65%)};
+  --ion-text-color-step-700: #{mix($background-color-value, $text-color-value, 70%)};
+  --ion-text-color-step-750: #{mix($background-color-value, $text-color-value, 75%)};
+  --ion-text-color-step-800: #{mix($background-color-value, $text-color-value, 80%)};
+  --ion-text-color-step-850: #{mix($background-color-value, $text-color-value, 85%)};
+  --ion-text-color-step-900: #{mix($background-color-value, $text-color-value, 90%)};
+  --ion-text-color-step-950: #{mix($background-color-value, $text-color-value, 95%)};
+}
+
 // Default Foreground and Background Step Colors
 // -------------------------------------------------------------------------------------------
 // Color Steps are used to provide variations in text and background colors of elements.
```

Comment:
Do we need to keep the `mix` fallbacks here? These will never be used because we are defining the step colors. 

```
$background-color-step-50:          var(--ion-background-color-step-50, var(--ion-color-step-50, mix($text-color-value, $background-color-value, 5%))) !default;
```

Reply:
I'm okay removing them, but I'm also not 100% confident there isn't some context I'm missing re: why they were there in the first place, considering we were also defining `--ion-color-step-#` already before.